README.md

```diff
@@ -130,15 +130,15 @@ message = "Error correction {}".format(outcome)
 print(message)
 ```
 
-See our [Documentation](https://flamingpy.readthedocs.io/en/latest/) for more tutorials.
+See our [Documentation](https://flamingpy.readthedocs.io/en/latest/usage/tutorials.html) for more tutorials.
 
 <!-- ## Performance Demos
   -->
 
 
 ## Contribution
 
-We welcome new contributions -- simply fork the FlamingPy repository and make a pull request (PR) containing your contribution. All contributors to FlamingPy will be listed as authors on the releases. Users who contribute significantly to the code (new plugins, functionalities, etc.) may be listed on the arXiv preprints for FlamingPy. See our [release notes and changelogs](https://github.com/XanaduAI/flamingpy/releases) for more details.
+We welcome new contributions -- simply fork the FlamingPy repository and make a pull request (PR) containing your contribution. All contributors to FlamingPy will be listed as authors on the changelog and releases. Users who contribute significantly to the code (new plugins, functionalities, etc.) may be listed on the arXiv preprints for FlamingPy. See our [`CHANGELOG.md`](.github/CHANGELOG.md) and [release notes](https://github.com/XanaduAI/flamingpy/releases) for details and you can always reach us on FlamingPy [Discussions](https://github.com/XanaduAI/flamingpy/discussions).

```

Comment:
Instead of having several copies of this contribution section, we should instead include the `.github/CONTRIBUTING.md` file.
```suggestion
.. include:: ../.github/CONTRIBUTING.md
```

(I believe this might work)

Reply:
Yup, great suggestion, I have a ticket to do this for a couple of files, but why not we implement it in this PR for CONTRIBUTING.md as you did.

Reply:
If everyone is happy with this solution, I have to later change to absolute links (rather than relative paths), so PyPI works as well.